digiwf-integrations/digiwf-example-integration/digiwf-example-integration-starter/src/main/java/de/muenchen/oss/digiwf/example/integration/ExampleIntegrationAutoconfiguration.java

```diff
@@ -1,43 +1,30 @@
 package de.muenchen.oss.digiwf.example.integration;
 
 
```

Comment:
Hier ist eine komische Leerzeile.

```suggestion
package de.muenchen.oss.digiwf.example.integration;

```

Reply:
Habs korrigiert, glaub da ist die ein Fehler in den Change-Vorschlag reingerutscht